en/drafts/originals/clustering-visualizing-word-embeddings.md

```diff
@@ -707,25 +702,25 @@ for ddc_name in sorted(clustered_df[f'ddc{ddc_level}'].unique()):
+With four DDCs and four clusters we have 16 plots in total. While the C-TF/IDF plots are not in and of themselves conclusive with respect to the assignment of any _one_ thesis, they do help us to get to grips with the aggregate differences; for documents in each DDC2 class, we're looking at *how* the vocabularies in the documents that were 'misclustered' with documents from another DDC2 differ from the vocabulary of the set that were 'correctly' clustered (which we define here as the modal cluster). This allows us to see how their contents (as viewed through the lens of TF/IDF) differ from the main cluster into which documents with their DDC were clustered.
 
-{% include figure.html filename="doc_vec-umap-d4-ddc2-c4-class_tfidf-History of ancient world (to c. 499).png" caption="'Misclassified' theses from the History of the ancient world (to c.499) DDC" %}
+{% include figure.html filename="doc_vec-umap-d4-ddc2-c4-class_tfidf-History of ancient world (to c. 499).png" alt="Visual description of figure image" caption="Figure 5. 'Misclassified' theses from the History of the Ancient World (to c.499) DDC" %}
 
-{% include figure.html filename="doc_vec-umap-d4-ddc2-c4-class_tfidf-History.png" caption="'Misclassified' theses from the History DDC" %}
+{% include figure.html filename="doc_vec-umap-d4-ddc2-c4-class_tfidf-History.png" alt="Visual description of figure image" caption="Figure 6. 'Misclassified' theses from the History DDC" %}
 
-For instance, taking the two history DDCs we can see that documents clustered with Linguistics seem to have an educational component, while those clustered with Philosophy from History of the Ancient World reveal terms associated with Greek philosophy and those from more general History appear to have a strong 17th and 18th Century element. From what we can see of the misclustered History and Ancient History theses it's reasonable to infer a subtle, but meaningful difference between concerns with history as one of objects and sites, and one more focussed in issues of power, work, politics and empire.
+For instance, taking the two History DDCs we can see that documents clustered with Linguistics seem to have an educational component, while those clustered with Philosophy from History of the Ancient World reveal terms associated with Greek philosophy and those from more general History appear to have a strong seventeenth and eighteenth century element. From what we can see of the misclustered History and Ancient History theses it's reasonable to infer a subtle, but meaningful difference between concerns with history as one of objects and sites, and one more focussed in issues of power, work, politics and empire.
```

Comment:
Our style guide [advises use of lower case for university departments](https://programminghistorian.org/en/author-guidelines#capitalisation). Are the capital letters critical for this aspect of the clustering?